using-mobile-extensions/adobe-journey-optimizer/README.md

```diff
@@ -159,13 +159,13 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 {% endtab %}
 {% endtabs %}
 
-#### Synch user push token with Adobe
+#### Sync the user push token with Adobe
 
-Use the setPushIdentifier API to sync user push token from the device with Adobe Experience Platform services.
+After importing and registering the extensions with your application, you need to sync the push token with Adobe Experience Platform, by using the setPushIdentifier API.
 
 {% tabs %}
 {% tab title="Android" %}
-To retrieve the push token from Firebase Messaging Service follow this [Firebase documentation](https://firebase.google.com/docs/cloud-messaging/android/client#retrieve-the-current-registration-token). After retrieving the push token use the below core API to sync it with profile in platform.
+To retrieve the push token from Firebase Messaging Service, please read the tutorial on [retrieving the registration token] within the Firebase documentation(https://firebase.google.com/docs/cloud-messaging/android/client#retrieve-the-current-registration-token). After retrieving the push token, you can use the following API to sync it with Profile in Platform.
```

Comment:
for the link `retrieving the registration token`, the actual link `https://firebase.google.com/docs/cloud-messaging/android/client#retrieve-the-current-registration-token` should be next to the link text.